sdk/storage/storage-blob/execute-samples.js

```diff
@@ -0,0 +1,76 @@
+const execa = require("execa");
+const fs = require("fs");
+require("dotenv").config({ path: "../.env" });
+
+// Samples can be skipped by mentioning them in the skipSet
+const skipSet = ["proxyAuth"];
+
+const bDel = `!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!`;
+const del = `${bDel}------------------------------${bDel}`;
+
+const { g, y, r, b } = [["r", 1], ["g", 2], ["b", 4], ["y", 3]].reduce(
+  (cols, col) => ({
+    ...cols,
+    [col[0]]: (f) => `\x1b[3${col[1]}m${f}\x1b[0m`
+  }),
+  {}
+);
+
+async function exec(cmd, cwd) {
+  let command = execa(cmd, {
+    cwd,
+    shell: true
+  });
+  command.stderr.pipe(process.stderr);
+  command.stdout.pipe(process.stdout);
+  return command;
+}
+
+async function runSamples(language) {
+  let cmd;
+  const directory = `./samples/${language}`;
+
+  if (language === "typescript") {
+    cmd = "ts-node";
+  } else {
+    cmd = "node";
+    await exec(`npm run build:js-samples`, directory);
```

Comment:
Nit: The script `build:js-samples` should have a better name :)

The `build:js-samples` script is for building the source code such that the js samples can be run right? I spent a few seconds wondering why do we need to build javascript samples :)

Reply:
`build:js-samples` -> `bundle:src`, sounds good?

Reply:
Also, there are other references to this `build:js-samples` script, VS Code debug config uses this  `build:js-samples` as a pre-launch task to allow debugging the samples.
Should I rename all these references in storage-packages in this same PR?

Reply:
Nah, create an issue for it, we can get to it next week